src/m3ninx/index/segment/builder/builder.go

```diff
@@ -23,18 +23,38 @@ package builder
 import (
 	"errors"
 	"fmt"
+	"sync"
 
 	"github.com/m3db/m3/src/m3ninx/doc"
 	"github.com/m3db/m3/src/m3ninx/index"
 	"github.com/m3db/m3/src/m3ninx/index/segment"
 	"github.com/m3db/m3/src/m3ninx/postings"
 	"github.com/m3db/m3/src/m3ninx/util"
+	"go.uber.org/atomic"
```

Comment:
nit: This should be grouped next to other third party imports such as `"github.com/cespare/xxhash"`